internal/painter/gl/draw.go

```diff
@@ -97,12 +96,111 @@ func (p *painter) drawRaster(img *canvas.Raster, pos fyne.Position, frame fyne.S
 	p.drawTextureWithDetails(img, p.newGlRasterTexture, pos, img.Size(), frame, canvas.ImageFillStretch, float32(img.Alpha()), 0)
 }
 
-func (p *painter) drawRectangle(rect *canvas.Rectangle, pos fyne.Position, frame fyne.Size) {
-	if (rect.FillColor == color.Transparent || rect.FillColor == nil) && (rect.StrokeColor == color.Transparent || rect.StrokeColor == nil || rect.StrokeWidth == 0) {
-		return
+func (p *painter) drawRectangle(
+	rect *canvas.Rectangle,
+	pos fyne.Position,
+	frame fyne.Size,
+) {
```

Comment:
I don't think we really do this with the input parameters. Would prefer if you left them on one line like before.